linux_os/guide/system/bootloader-grub2/grub2_pti_argument/rule.yml

```diff
@@ -0,0 +1,56 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Enable Kernel page-table isolation'
```

Comment:
```suggestion
title: 'Enable Kernel Page-Table Isolation (KPTI)'
```

Reply:
Many will know this by PTI or KPTI. The intent of the check may be easier to understand when the acronym is included in parenthesis.

Reply:
Updated in eba83009dcecfdba01d9b46668431f49bc1ffc40.